scanpipe/views.py

```diff
@@ -102,14 +102,22 @@ class ProjectListView(
     filterset_class = ProjectFilterSet
     template_name = "scanpipe/project_list.html"
     prefetch_related = ["runs"]
-    paginate_by = 10
+    paginate_by = 20
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["active_count"] = Project.objects.filter(is_archived=False).count()
         context["archived_count"] = Project.objects.filter(is_archived=True).count()
         return context
 
+    def get_ordering(self):
+        ordering = self.request.GET.get("sort")
+        return (
+            ordering
```

Comment:
This needs to be implemented on the `ProjectFilterSet` instead.
See https://django-filter.readthedocs.io/en/main/ref/filters.html?highlight=OrderingFilter#orderingfilter

Reply:
Is this enough for ProjectFilterSet?

```
sort = django_filters.OrderingFilter(
   fields=(
       ("name", "name"),
       ("created_date", "created_date")
    ),
)
```

Reply:
Yes it, works

Reply:
sure, this part of the code works and I have added it to this [commit](https://github.com/nexB/scancode.io/pull/446/commits/042c70d20f8791d7d8ad8851445311b5c65805c2)